ako-crd-operator/internal/controller/routebackendextension_controller_test.go

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
This file will have to be replaced with UTs

Reply:
will be taken in another pr.